install.sh

```diff
@@ -425,8 +323,8 @@ do_install() {
 				set -x
 				$sh_c 'apt-get update'
 				$sh_c "apt-get install -y -q $pre_reqs"
-				curl -fsSl "https://download.docker.com/linux/$lsb_dist/gpg" | $sh_c 'apt-key add -'
-				$sh_c "add-apt-repository \"$apt_repo\""
+				curl "https://download.docker.com/linux/$lsb_dist/gpg" | $sh_c 'apt-key add -'
+				$sh_c "echo \"$apt_repo\" > /etc/apt/sources.list.d/docker.list"
```

Comment:
meant as a simple install script for newbie users to quickly install docker on a new system, we don't need to handle all cases of migration process